### PR TITLE
Fixes part of #4178 : Added dark mode support to OnboardingActivity

### PR DESCRIPTION
**Pull request by bhaktideshmukh**
## Explanation
Fixes part of #4178 : Added dark mode support to OnboardingActivity

Mocks link: 

- https://xd.adobe.com/view/c05e9343-60f6-4c11-84ac-c756b75b940f-950d/screen/5981ecd6-2a80-4d82-8715-3e92294d53b5/specs/
- https://xd.adobe.com/view/c05e9343-60f6-4c11-84ac-c756b75b940f-950d/screen/69f9affc-7eed-4075-84a5-22d3536c6056/specs/
- https://xd.adobe.com/view/c05e9343-60f6-4c11-84ac-c756b75b940f-950d/screen/9caedbc6-7055-4cff-88e6-caf0052bcf0b/specs/
- https://xd.adobe.com/view/c05e9343-60f6-4c11-84ac-c756b75b940f-950d/screen/7aa006a4-26e4-4525-9609-69795411159a/specs/

**Screenshots**
| Default  | Dark mode|
|-------|---|
|||

| Default | Dark mode |
|-------|---|
|||

| Default | Dark mode |
|-------|---|
|||

| Default  | Dark mode |
|-------|---|
|||

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Comment by bhaktideshmukh** [comment]
@rt4914 
One doubt -- here in slide number 3 [link](https://xd.adobe.com/view/c05e9343-60f6-4c11-84ac-c756b75b940f-950d/screen/9caedbc6-7055-4cff-88e6-caf0052bcf0b/specs/) of onboarding, different color is used in mocks for dark mode but in default mode 2 different shades are used for portrait and landscape mode in download sign figure.
So please suggest whether it should be changed or not?

**Comment by rt4914** [comment]
> 

@bhaktideshmukh I am unable to understand what exactly you mean here? 
**different color is used in mocks for dark mode ** -> Can you precise what you are taking about? The background, icon, text, etc?
**default mode 2 different shades are used** -> What is default mode here? Also what are these two different shades and are they used in? TextView, image or background? Also if possible share links of these two different things that you are talking about.

**Comment by bhaktideshmukh** [comment]
> > 
> 
> @bhaktideshmukh I am unable to understand what exactly you mean here? **different color is used in mocks for dark mode ** -> Can you precise what you are taking about? The background, icon, text, etc? **default mode 2 different shades are used** -> What is default mode here? Also what are these two different shades and are they used in? TextView, image or background? Also if possible share links of these two different things that you are talking about.

Here in ic_portrait_onboarding_2.xml the fillColor used is #00645c where as in ic_landscape_onboarding_2.xml and ic_landscape_onboarding_2_tablet.xml it is #02645c.
And the color used in [mocks](https://xd.adobe.com/view/c05e9343-60f6-4c11-84ac-c756b75b940f-950d/screen/9caedbc6-7055-4cff-88e6-caf0052bcf0b/specs/) is #3BD1C4.

**Comment by rt4914** [comment]
> 

@bhaktideshmukh Do one thing here.

File one detailed issue mentioned this issue and for now use one single color only for dark + light mode which is `ic_portrait_onboarding_2.xml`.

Once you have filed the issue, mention it here also. And share with me so that I will add more details and then I will share it with design team so that they can clarify things. That way your PR won't get blocked and also it can be a nice goo-first-issue too.

**Comment by bhaktideshmukh** [comment]
> > 
> 
> @bhaktideshmukh Do one thing here.
> 
> File one detailed issue mentioned this issue and for now use one single color only for dark + light mode which is `ic_portrait_onboarding_2.xml`.
> 
> Once you have filed the issue, mention it here also. And share with me so that I will add more details and then I will share it with design team so that they can clarify things. That way your PR won't get blocked and also it can be a nice goo-first-issue too.

I have filed the issue regarding this and have assigned you as well
https://github.com/oppia/oppia-android/issues/4381
And have changed the fillColor of `ic_landscape_onboarding_2.xml` and `ic_landscape_onboarding_2_tablet.xml` to the fillColor of `ic_portrait_onboarding_2.xml`  .

File: app/src/main/res/drawable/ic_landscape_onboarding_2.xml [hunk 1]
```diff
@@ -35,5 +35,5 @@
   </path>
   <path
       android:pathData="M202.2,157.8H167V105H114.2v52.8H79l61.6,61.6ZM79,237v17.6H202.2V237Z"
-      android:fillColor="#02655c"/>
+      android:fillColor="#00655c"/>
```
Comment (rt4914) [inline app/src/main/res/drawable/ic_landscape_onboarding_2.xml]: Don't update this color code. I have changed the description of your issue and therefore its fine if its inconsistent right now.

File: app/src/main/res/drawable/ic_landscape_onboarding_2_tablet.xml [hunk 2]
```diff
@@ -42,6 +42,6 @@
         android:pathData="M0,-12h320v360h-320z"/>
     <path
         android:pathData="M221.6,134h-35.2V81.2h-52.8V134H98.4l61.6,61.6L221.6,134zM98.4,213.2v17.6h123.2v-17.6H98.4z"
-        android:fillColor="#02655C"/>
+        android:fillColor="#00655C"/>
```
Comment (rt4914) [inline app/src/main/res/drawable/ic_landscape_onboarding_2_tablet.xml]: Don't update this color code. I have changed the description of your issue and therefore its fine if its inconsistent right now.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@bhaktideshmukh Final nit change. Earlier I did suggest these changes but lets first get the confirmation from design team that the images cannot be of different colors.

File: app/src/main/res/drawable/ic_landscape_onboarding_2.xml (hunk 1 above)
Comment (bhaktideshmukh) [reply app/src/main/res/drawable/ic_landscape_onboarding_2.xml]: Done

File: app/src/main/res/drawable/ic_landscape_onboarding_2_tablet.xml (hunk 2 above)
Comment (bhaktideshmukh) [reply app/src/main/res/drawable/ic_landscape_onboarding_2_tablet.xml]: updated

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**